backend/.env

```diff
@@ -0,0 +1 @@
+MONGODB_URI=mongodb://localhost:27017/local
```

Comment:
.env file is usually not checked in to the repo because it might contains secrets. It's better to check in template file (.env.template) and whomever that run the system will set the right env (in our case the Github action set it right before running the script)